com.ibm.wala.util/src/main/java/com/ibm/wala/util/collections/AbstractMultiMap.java

```diff
@@ -82,7 +83,7 @@ public Set<V> get(K key) {
    * @see AAA.util.MultiMap#put(K, V)
    */
   @Override
-  public boolean put(K key, V val) {
+  public boolean put(K key, @Nullable V val) {
```

Comment:
This is another case where I would have made a different decision; I don't think we want to allow nulls here (though in this case I'm actually not sure, maybe some code relies on that